### PR TITLE
fix(detail page): Display saving text during work item creation.

### PR DESCRIPTION
**Pull request by nimishamukherjee**
fix(detail page): Display saving text during work item creation.

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
@mindreeper2420 , @michaelkleinhenz let me know if this PR helps better the user experience.
As there is a delay on prod-preview when saving a work item, we could show a Saving work item message.

**Comment by mindreeper2420** [comment]
I'll take a look at this tonight and let you know.

File: src/app/work-item/work-item-detail/work-item-detail.component.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <div [@slideInOut]="panelState" class="detail-panel">
+  <h3 *ngIf="saving">
+    <p><span class="spinner spinner-xs spinner-inline"></span>Saving work item.</p>
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/work-item-detail.component.html]: change the spinner to `spinner-sm` and the text to 'Saving Work Item...'

**Comment by mindreeper2420** [comment]
I like this 😄  I requested a change to the text and to make the spinner larger - that way it will be more noticeable. I'd also suggest that the message be moved to be in-line with the Work Item type icon (where the Title would eventually reside). This will prevent a jarring effect of pushing the items down when the message pops up.

**Pull request merged**